src/main/scala/org/clulab/reach/extern/export/fries/FriesOutput.scala

```diff
@@ -321,205 +323,209 @@ class FriesOutput extends JsonOutputter {
       if (contextId.isEmpty) {           // if this is a new context
         val ctxid = mkContextId(paperId, passage, mention.sentence) // generate new context ID
         contextIdMap.put(context, ctxid)      // save the new context ID keyed by the context
-        contextFrame = Some(mkContextFrame(paperId, passage, mention, ctxid, context))
+        contextFrame = Some(makeContextFrame(paperId, passage, mention, ctxid, context))
         contextId = Some(ctxid)               // save the new context ID
       }
     }
 
     (contextId, contextFrame)
   }
 
+
+  /** Lookup the unique ID, for the given mention, in the given maps, based on the type
+      of the mention. Returns an option for the ID found, or None if no ID found. */
   private def lookupMentionId (mention: Mention,
-                               entityMap:IDed,
-                               eventMap:IDed): Option[String] = mention match {
+                               entityMap: IDed,
+                               eventMap: IDed): Option[String] = mention match {
     case tbm: TextBoundMention => entityMap.get(tbm)
     case evm: EventMention => eventMap.get(evm)
+    case rem: RelationMention => eventMap.get(rem)
```

Comment:
Wouldn't something like `case m: Mention => entityMap.getOrElse(m, buildMePlease(m))` work for all three cases?